woocommerce/woocommerce-gutenberg-products-block
# Remove bullet points and unnecessary padding from `SearchListControl`

## What

On certain themes (such as TT1 and TT4) the `SearchListControl` used in blocks such as Single Product, Featured Product and Featured Category, might display traditional bullet list style and padding. This PR fixes that.

Fixes #11327, #10348
References #11299

## Testing Instructions

1. Add a Single Product, Featured Product and Featured Category to a page.
2. Ensure that the list of possible selection is shown as a list without bullet points and extra padding (the bottom border should reach all the way through).


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|     
<img width="639" alt="276505401-a13ba8e6-9e62-42b7-b567-d5b480fd2298" src="https://github.com/woocommerce/woocommerce-blocks/assets/1847066/9a528692-0bd4-4194-bc18-ec3f4623cf77"> |  <img width="642" alt="Screenshot 2023-10-25 at 17 57 36" src="https://github.com/woocommerce/woocommerce-blocks/assets/1847066/ae8ce0fe-c950-4e49-9ffb-825e12a6a0a0"> |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> `SearchListControl` doesn't show unnecessary bullet points in the editor view.
